synfig-core/src/modules/mod_ffmpeg/trgt_ffmpeg.cpp

```diff
@@ -70,7 +75,6 @@ SYNFIG_TARGET_SET_VERSION(ffmpeg_trgt,"0.1");
 /* === M E T H O D S ======================================================= */
 
 ffmpeg_trgt::ffmpeg_trgt(const char *Filename, const synfig::TargetParam &params):
-	pid(-1),
```

Comment:
is it ok to leave it uninitialized for non-MSVC?

Reply:
Yes. MSVC does not have `fork()` function, so this field is not used. I disabled it in the header file:

**trgt_ffmpeg.h**
```
#ifdef HAVE_FORK
	pid_t pid = -1;
#endif
```

Reply:
I moved the initialization into a header file: 
```
#ifdef HAVE_FORK
	pid_t pid = -1;
#endif
```